src/components/builds.js

```diff
@@ -1,16 +1,51 @@
 import React from "react"
+import PropTypes from "prop-types"
+import { HeaderLineBelow } from "../styles/Header"
+import styled from "styled-components"
+import Project from "./project.js"
 import Section from "styles/Section"
 
-const Builds = () => (
+const ProjectContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+
+  @media (min-width: 1000px) {
+    justify-content: space-between;
+  }
+`
+
+const Builds = ({ title, projects }) => (
   <Section>
-    <h1>Sandbox Builds...</h1>
-    <p>
-      We'll make some big ol' huge mountains! There's too many complicated
-      things in our life already. All you're worried about is this nice outside
-      shape. Always follow the angles. Don't just cover it all up -- leave these
-      spots!
-    </p>
+    <HeaderLineBelow>
+      <span>{title}</span>
+    </HeaderLineBelow>
+    <ProjectContainer>
+      {projects.map(p => (
+        <Project
```

Comment:
You should pass a `key` prop to each child of a `map` operation in React so that it knows how to properly identify each one. Basically it should just be some unique value associated with that specific item in the list, so if the order changes, React can still efficiently re-render it. `key={p.title}` here should work.